include/ignition/rendering/base/BaseGpuRays.hh

```diff
@@ -151,6 +151,9 @@ namespace ignition
       // Documentation inherited.
       public: virtual double VerticalResolution() const override;
 
+      // Documentation inherited.
+      public: virtual void SetScanningPattern(ScanningPattern pattern) override;
```

Comment:
`pattern` here is a non-const, but in the parent class (`GpuRays`)it is a const. Also in the definition (line 458) it is a const. Should it be const too here?

Reply:
Modified in the next commit.